app/scripts/modules/dcos/serverGroup/configure/CommandBuilder.js

```diff
@@ -124,6 +124,8 @@ module.exports = angular
     }
 
     function buildNewServerGroupCommand(application, defaults = {}) {
+      defaults = defaults || {};
+
```

Comment:
this shouldn't actually be doing anything, since the function signature already sets it to `{}` if it's null?

Reply:
hmmm idk, somehow defaults was getting to be null without it. I admit, I know very little about javascript.

Reply:
If you explicitly pass a falsey value, then the default parameter isn't applied.  i.e.:
```
buildNewServerGroupCommand(application, null);
```
I suggest removing the default parameter on line 126 and keep the reassignment on line 127.

Reply:
but by removing the parameter entirely, wouldn't that be a non-passive change? LIke, then you wouldn't have the option of not specifying the second parameter.

Reply:
I mean just removing the default value in the parameter declaration and leaving it as `buildNewServerGroupCommand(application, defaults)`.   In javascript you can omit arguments when calling a function. In TypeScript, the argument would have to be explicitly declared as optional, but this file is `.js`